src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/requestaccess/CreateDataAccessSubmissionStep2.java

```diff
@@ -115,6 +121,15 @@ public void invoke(SynapseSuggestion suggestion) {
 				CreateDataAccessSubmissionStep2.this.accessorsList.addUserBadge(suggestion.getId());
 			};
 		});
+		accessorsList.setUserIdsDeletedCallback(new CallbackP<List<String>>() {
+			@Override
+			public void invoke(List<String> param) {
+				if (dataAccessRequest instanceof DataAccessRenewal) {
```

Comment:
I think the right check would be if there is an approved submission submitted by the user for the given access requirement including the accessor?